config/config.exs

```diff
@@ -23,7 +23,7 @@ config :livebook,
   app_service_name: nil,
   app_service_url: nil,
   authentication_mode: :token,
-  explore_notebooks: [],
+  learn_notebooks: [],
```

Comment:
@josevalim do we deprecate or keep as explore?

Reply:
@fhunleth once Livebook v0.7 is out (next week), you will need to do this tiny renaming. :)